### ISSUE TITLE
long array countType in nbt.json should be an i32, not an i64

### ISSUE THREAD

**Issue by SliceThePi** [PrismarineJS/prismarine-nbt#22]
A long array's length is a 32-bit int, not a 64-bit int.

**Comment by rom1504** [comment]
No a long is a i64

**Comment by SliceThePi** [comment]
That's correct, but I'm talking about the number that's *counting* the longs. It's an array of up to (2^31-1) 64-bit ints.

**Comment by rom1504** [comment]
indeed you're right (reference https://wiki.vg/NBT#Specification)
fixed and released as 1.2.1
